features/step_definitions/navigation_steps.rb

```diff
@@ -130,6 +130,17 @@ def find_record_for(object, schema, name)
   click_link(link)
 end
 
+When /^I click "(.*)" (.*) link$/ do |link, position|
```

Comment:
maybe we should be using `within` to isolate the clicking on the breadcrumb via it's id?